src/test/java/org/example/c06challenges/ArmstrongNumbersTest.java

```diff
@@ -7,13 +7,13 @@
 class ArmstrongNumbersTest {
     @Test
     void returnTrueIfNumberIsArmstrongNumber() {
-        int number = 1;
+        int number = 4;
```

Comment:
Test against some of this numbers.
The Armstrong numbers of first kind up to 10 digits are 1, 2, 3, 4, 5, 6, 7, 8, 9, 153, 370, 371, 407, 1634, 8208, 9474, 54748, 92727, 93084, 548834, 1741725, 4210818, 9800817, 9926315, 24678050, 24678051, 88593477, 146511208, 472335975, 534494836, 912985153, and 4679307774.